DEVELOPER.md

```diff
@@ -99,24 +99,18 @@ make build/kumactl
 export PATH=`pwd`/build/artifacts/kumactl:$PATH
 ```
 
-2. Configure a `kumactl` with running Control Plane
-
-```bash
-kumactl config control-planes add --name universal --address http://localhost:5681
-```
-
-3. Apply a Dataplane descriptor
+2. Apply a Dataplane descriptor
```

Comment:
This is a Developer Guide.

Why hide `kumactl config control-planes add` away from developers ?

I would rather suggested a user(developer)
1.  to run `kumactl config control-planes list` 
2. and if the actual configuration is different from what we're expecting, then run `kumactl config control-planes add --name local --address http://localhost:5681`